src/App.css

```
@@ -19,7 +19,17 @@ ul {
   list-style: none;
 }
 
+.container {
+  text-align: center;
+  overflow: auto;
+  margin-top: 50px;
+  padding: 20px;
+}
+
 .content-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   height: calc(100vh - 100px);
```

We know the CSS styles aren't important these weeks. However, I would like to say only as a comment. What do you think about [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)?

```suggestion
/*.....*/
// css variables
:root {
  --height-element: 100px;
}

  height: calc(100vh - var(--height-element));
```

or I think it could be much better:

```suggestion
:root {
  --height-element: 100px;
  --height-content-container: calc(100vh - var(--height-element));
}

.content-container{
  /*...*/
  height: var(--height-content-container);
}
```